[140745] Migrate to Chrome Manifest v3

lcao1602 opened the pull request:
I used jsonRPC instead of xmlRPC to handle the fetch requests. I also adapted the event handlers to grab the correct data from the response.

jtl3d commented on background.js:
```diff
@@ -1,11 +1,10 @@
+try {
```
test line comment

jtl3d reviewed (commented):
Test review w/time

jtl3d commented:
test comment

jtl3d commented on src/bugzilla.js:
```diff
@@ -93,190 +95,194 @@ Bugzilla.prototype.getBugs = function(bugIds, includeFields) {
-	});
-}
+
+	const methodName = "User.logout";
```
This should be Bug.fields

jtl3d reviewed (changes requested):
Milestones don't work and also the popup with the products drop down... I think it's because of this line.

lcao1602 commented:
I fixed the typo with the milestone tab. I manage to get the login form to appear, but right now it is not invoking the correct callback functions.

lcao1602 commented:
I have addressed all the issues that we had with the functionality of the extension.

The only issue left is the error `Uncaught (in promise) Error: The message port closed before a response was received.` which is possibly caused by how the background and content-script are setup to communicate with one another. 

However, the error is just console noise and does not affect the functionalities.

lcao1602 commented:
test comment

lcao1602 commented:
test comment with time
